<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDocumentDialog</class>
 <widget class="QDialog" name="NewDocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Document</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Image size:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="imageWidthEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="imageWidthUnitBox">
        <item>
         <property name="text">
          <string>px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>in</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ã—</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imageHeightEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="imageHeightUnitBox">
        <item>
         <property name="text">
          <string>px</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>in</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Preset:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Image resolution:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="imageResolutionEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>pixels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="imageResolutionUnitBox">
        <item>
         <property name="text">
          <string>per in</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>per cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>per mm</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Color model:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="colorModelBox">
        <item>
         <property name="text">
          <string>RGB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CMYK</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>at</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bitDepthBox">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>bits per channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageWidthEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>widthChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageHeightEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>heightChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageResolutionEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>resolutionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageWidthUnitBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>widthUnitChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageHeightUnitBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>heightUnitChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageResolutionUnitBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>resolutionUnitChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>widthChanged()</slot>
  <slot>heightChanged()</slot>
  <slot>widthUnitChanged()</slot>
  <slot>heightUnitChanged()</slot>
  <slot>resolutionChanged()</slot>
  <slot>resolutionUnitChanged()</slot>
 </slots>
</ui>
